apps/taskman/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class ScheduledClosingTask(models.Model):
+    """This instance represents when should a Jira task should be closed if there is no changes"""
```

Comment:
nitpick: double `should`
nitpick: `is` -> `are`

Reply:
Thank you, submitting a fix for that.